ios/Sources/SettingFeature/SettingScreen.swift

```diff
@@ -0,0 +1,71 @@
+import Styleguide
+import SwiftUI
+import UIKit
+
+public struct SettingToggleModel: Hashable {
```

Comment:
It seems better to create this with enum.

Reply:
@ostk0069 
I make `SettingModel` enum type and create `toggle` case. 

By doing this, binding way is a little changed, so please tell me your idea!

Reply:
What I was thinking is like this.
If you have any problem with this, please let me know.

```
public enum SettingItem {
    case darkMode(isOn: Bool)
    case language(isOn: Bool)

    var title: String {
        switch self {
        case .darkMode:
            return L10n.SettingScreen.ListItem.darkMode
        case .language:
            return L10n.SettingScreen.ListItem.language
        }
    }
}
```

Reply:
I agree with it because all of SettingItem are toggle type and my way was redundant.

↓Updated
```swift
public enum SettingModel: Hashable {
    case darkMode(isOn: Bool)
    case language(isOn: Bool)
    
    var title: String {
        switch self {
        case .darkMode:
            return L10n.SettingScreen.ListItem.darkMode
        case .language:
            return L10n.SettingScreen.ListItem.language
        }
    }
    
    var isOn: Bool {
        switch self {
        case let .darkMode(isOn), let .language(isOn):
            return isOn
        }
    }
    
    mutating func update(isOn: Bool) {
        switch self {
        case .darkMode:
            self = .darkMode(isOn: isOn)
        case .language:
            self = .language(isOn: isOn)
        }
    }
}
```